.github/workflows/llvm-project.yml

```diff
@@ -0,0 +1,31 @@
+name: Build kernel.org toolchain
+
+on: workflow_dispatch
+jobs:
+  stage_1:
+    name: stage 1
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        image: [llvm-project]
```

Comment:
The title of this workflow makes it seem like this is only going to be used for `llvm-project`, so get rid of the matrix and sink it into the `Build` step below?

Reply:
we might want to expand this to musl, linux, etc for stage 3, but I'll do that when we get there.
done PTAL

Reply:
Ack. Regardless, I suspect we will want those image builds in their own job flow anyways.